core/src/main/scala/org/elasticmq/actor/queue/QueuePersister.scala

```diff
@@ -0,0 +1,11 @@
+package org.elasticmq.actor.queue
+
+import org.elasticmq.QueueData
+
+trait QueuePersister {
```

Comment:
Maybe it would be easier if there was a `QueuesMetadataListener`, which would simply receive the list of current queues (the `queues` from `QueueManagerActor`) every time anything changes in any queue. We wouldn't have to maintain any state in the listener/persister implementation, and there would be only one method here

Reply:
Ah ok not so easy, as we don't really have that list anywhere. Then we'll have to keep a `CHM` or sth like that in the implementation, but still I think the listener approach would be more appropriate, as the queue actors don't really have to care about persistence. Simple `QueueMetadataListener.[created | updated | remove]` would be enough?